mule-maven-plugin/pom.xml

```diff
@@ -80,6 +86,14 @@
         </dependency>
     </dependencies>
 
+    <repositories>
+        <repository>
+            <id>mule-releases</id>
+            <name>Mule Release Repository</name>
+            <url>https://repository-master.mulesoft.org/nexus/content/repositories/releases</url>
+        </repository>
+    </repositories>
```

Comment:
just to check it hasn't been added by accident

Reply:
You were right, not needed.